### ISSUE TITLE
Does not work for IM tabs

### ISSUE THREAD

**Issue by zevelorgani** [EionRobb/pidgin-ignore-nickchange#6]
Every time I start Pidgin, my open IM chats get several "XXX is now known as YYY" messages.
I installed the plugin, set both parameters to "1" and checked the "IM" option, still happening.

Please advise.

**Comment by EionRobb** [comment]
Good point, the plugin was only intended for multi-user chats, but it could be adapted for one-to-one IM's too

**Comment by zevelorgani** [comment]
Apparently a known problem:
https://developer.pidgin.im/ticket/8630

**Comment by EionRobb** [comment]
So once we add that to this plugin, then we can point that ticket here instead :)

**Comment by zevelorgani** [comment]
Does it matter if it's handled in a plugin or by the main program as long as it is fixed?
I'll beta-test it for you.

**Comment by EionRobb** [comment]
Doesn't really matter if it's done in a plugin, in my opinion.  I'll see what needs to happen in the plugin to make this happen :)

**Comment by alexolog** [comment]
Excellent!  Can you please put it in "releases" when ready?  It's easier to track changes that way.

**Comment by alexolog** [comment]
Are there any news on that one?

**Comment by EionRobb** [comment]
https://github.com/EionRobb/pidgin-ignore-nickchange/commit/c0b3e4a15e484ec5f8518e3ebbfb246cc31b78af should do it?  Can you lot please test? :)

[nickchange.zip](https://github.com/EionRobb/pidgin-ignore-nickchange/files/455760/nickchange.zip) is a dll of the plugin

**Comment by alexolog** [comment]
Looks good.

How do I set it up to only hide nick changes in IM conversations but not in multi-user chats?

**Comment by EionRobb** [comment]
Hmm good point.  There should probably be an option to not do it for chats.  I guess as a workaround setting 9999 for both the numeric inputs and just leaving the "Hide alias changes in IM messages" option checked.

**Comment by alexolog** [comment]
You mean 1000 and 480 (max values).
Incidentally, I have seen (and visited) rooms with over a 1000 people (granted, 90+% of them were silent).

**Comment by alexolog** [comment]
I still see the messages in (for instance) hangouts conversations when they are loaded.

> <ddddddddddddddd> is now known as <alias>.

**Comment by alexolog** [comment]
Is there a specific reason it works with some protocols but not with others (hangouts)?

**Comment by alexolog** [comment]
Does not work for hangouts and for skype

**Comment by alexolog** [comment]
Revisiting this since I am getting a *lot* of nick change messages lately on Hangouts.

**Comment by SpiderJonny** [comment]
Is there any way to get this working for Hangouts?   The "is now known as" messages are quite intrusive :(
